src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -114,7 +131,7 @@
 
         (:error-message @state) [:div {:style {:color "red"}}
                                  "FireCloud service returned error: " (:error-message @state)]
-        :else [comps/Spinner {:text "Loading configurations for import..."}])])})
+         :else [comps/Spinner {:text "Loading configurations for import..."}])])})
```

Comment:
I can line up the ":" with the "(" of ":error-message" if you want... but it seemed better to me to line up the ":" with each-other.